torch/__init__.py

```diff
@@ -30,6 +30,7 @@
 from ._six import string_classes as _string_classes
 
 from typing import Set, Type, TYPE_CHECKING
+from math import e , nan , inf , pi
```

Comment:
These should also be included in `__all__`. It probably makes sense to not have these imports at the top, but rather at the import where they are added to `__all__`.

Reply:
Hey @pmeier I`ve made the change , is it fine now ?

Reply:
I think we talked past each other. I'm advocating for one of two styles:

1. Have the `from math import ...` in the default `import`'s block at the top of the file. In that case you just need to add the constants to `__all__`.
2. The constants might have their own section in the file like this:

    ```python
    ################################################################################
    # Define numeric constants
    ################################################################################
    
    from math import e, pi, nan, inf
    __all__.extend(["e", "pi", "nan", "inf"])
    ```

    This keeps the import as well as our definition, i.e. just adding them to `__all__`, together which makes it easy to read.